package.json

```diff
@@ -27,7 +27,9 @@
   "license": "MIT",
   "node-red": {
     "nodes": {
-      "ttn": "ttn/ttn.js"
+      "ttn-app": "ttn-app/ttn-app.js",
+      "ttn": "ttn/ttn.js",
+      "ttn-downlink": "ttn-downlink/ttn-downlink.js"
```

Comment:
In line with other client libs I'd suggest we name this `ttn-send`. I moved away from `up/down` because the interpretation depends on your perspective, while `send` is clearly from the app's perspective.